youtube_dl/extractor/threeqsdn.py

```diff
@@ -132,11 +151,16 @@ def extract_formats(item_url, item={}):
 
         self._sort_formats(formats)
 
-        title = self._live_title(video_id) if live else video_id
+        webpage = self._download_webpage(self._API_URL + video_id, video_id)
```

Comment:
This will break the extraction if webpage download fails. Also, `title` and `description` from the webpage does not bring any senseless names so there is no point extracting them. Same title may be extracted from JS code.